deploy/s3/action.yml

```diff
@@ -0,0 +1,43 @@
+name: AWS Deploy S3 Files
+description: Deploy files to an s3 bucket
+inputs:
+  source:
+    description: "source directory to upload into the s3 bucket"
+    required: true
+    default: "./dist"
+  bucket:
+    description: bucket name to deploy to
+    required: true
+  prefix:
+    description: "prefix where to upload files in bucket"
+    default: ""
+    required: true
+  aws-role:
+    description: "aws role name"
+    required: true
+  aws-region:
+    description: "aws region"
+    required: true
+    default: us-east-1
+  invalidate-cloudfront: 
+    description: "invalidate cloudfront distribution"
+    required: true
+    default: "false"
+  cloudfront-distribution:
+    description: "cloudfront distribution id"
```

Comment:
You can maybe add that this is required when `invalidate-cloudfront` is set to true?

Reply:
I don't know if it is possible.

Reply:
I mean we can simply add this detail to the field's description for now. 

You could probably add this logic but it would require you to add code that uses GitHub's core library.